app/controllers/sessions_controller.rb

```diff
@@ -2,19 +2,47 @@ class SessionsController < Clearance::SessionsController
   def create
     @user = find_user(params.require(:session))
 
+    if mfa_enabled? && @user&.mfa_enabled?
+      session[:mfa_user] = @user.id
```

Comment:
Can we please continue using `handle` (`session[:who]`) to find users?

Reply:
You mean store `who` in session and find user with `who`, not `id`? That seems better in security.

Reply:
yes

Reply:
Since we should not store passwords in session, here we just needs to store `session[:who]`. But in `find_user`, `session` is just a param, not real sessions. So it seems that we do this to just hide user's id.